contrib/client/help.py

```diff
@@ -10,7 +10,7 @@ def show_usage():
         "\n\nGLOBAL OPTIONS:"
         "\n\t--apiconnect \tAPI server where to send the requests. Defaults to 'localhost' (modifiable in conf file)."
         "\n\t--apiport \tAPI port where to send the requests. Defaults to '9814' (modifiable in conf file)."
-        "\n\t-d, --debug \tShows debug information and stores it in teos_client.log."
+        "\n\t-d, --debug \tShows debug information and stores."
```

Comment:
En passant, so I don't forget: should remove `" and stores"`.

Reply:
I'm already working on a miscellaneous PR including that. Will push in a bit.